src/platform/windows/display_base.cpp

```diff
@@ -363,8 +364,10 @@ namespace platf::dxgi {
   // we spawn a helper tool to probe for us before we set our own GPU preference.
   bool
   probe_for_gpu_preference(const std::string &display_name) {
-    // If we've already been through here, there's nothing to do this time.
     static bool set_gpu_preference = false;
+    static bool verify_frame_capture = true;
```

Comment:
Let's just loop through both `verify_frame_capture == true` and `verify_frame_capture == false` cases here. We can't necessarily assume that the caller will invoke this function again.

Reply:
created new private method, adjusted original method to call both scenarios like suggested